core/templates/pages/topics-and-skills-dashboard-page/templates/assign-skill-to-topic-modal.template.html

```diff
@@ -6,6 +6,7 @@ <h3>Select topic(s) to which the skill is to be assigned</h3>
   <select-topics topic-summaries="topicSummaries"
                  selected-topic-ids="selectedTopicIds">
   </select-topics>
+  <em ng-if="!topicSummaries.length">There are no topics to assign this skill to.</em>
```

Comment:
Maybe "there are no additional topics" or "The skill is already assigned to all available topics"? Since, this would only occur if the skill is assigned to all topics in the server, right?

Reply:
done